test/helper/webapi.js

```diff
@@ -648,7 +648,8 @@ module.exports.tests = function () {
     });
   });
 
-  describe('#attachFile', () => {
+  describe.skip('#attachFile', () => {
```

Comment:
ups, this is going to skip this test for ALL helpers!
please check webapi.js there are examples how to skip tests for exactly one helper